frontend/cypress.json

```diff
@@ -7,5 +7,6 @@
   "screenshotsFolder": "mochawesome-report/assets",
   "trashAssetsBeforeRuns": false,
   "viewportWidth": 1366,
-  "viewportHeight": 768
+  "viewportHeight": 768,
+  "chromeWebSecurity": false
```

Comment:
This allows cypress to access iframe DOMs, which we need to do for the template e2e tests